diff --git b/html/NoAuth/css/jsgantt.css a/html/NoAuth/css/jsgantt.css
index 683fcad..5845df1 100755
--- b/html/NoAuth/css/jsgantt.css
+++ a/html/NoAuth/css/jsgantt.css
@@ -49,5 +49,5 @@
 
 DIV.scroll { BORDER-RIGHT: #efefef 1px solid; PADDING-RIGHT: 0px; BORDER-TOP: #efefef 1px solid; PADDING-LEFT: 0px; PADDING-BOTTOM: 0px; OVERFLOW: hidden; BORDER-LEFT: #efefef 1px solid; WIDTH: 420px; PADDING-TOP: 0px; BORDER-BOTTOM: #efefef 1px solid; BACKGROUND-COLOR: #ffffff }
 
-DIV.scroll2 { position:relative; PADDING-RIGHT: 0px; overflow:auto ;overflow-x:scroll;overflow-y:hidden; PADDING-LEFT: 0px; PADDING-BOTTOM: 0px; WIDTH: 482px; PADDING-TOP: 0px; BACKGROUND-COLOR: #ffffff }
+DIV.scroll2 { position:relative; PADDING-RIGHT: 0px; overflow:auto ;overflow-x:scroll;overflow-y:hidden; PADDING-LEFT: 0px; PADDING-BOTTOM: 0px; WIDTH: 720px; PADDING-TOP: 0px; BACKGROUND-COLOR: #ffffff }
 
diff --git b/html/NoAuth/js/jsgantt.js a/html/NoAuth/js/jsgantt.js
index fe99c57..8a9eaba 100755
--- b/html/NoAuth/js/jsgantt.js
+++ a/html/NoAuth/js/jsgantt.js
@@ -573,7 +573,11 @@ JSGantt.GanttChart =  function(pGanttVar, pDiv, pFormat)
 * @type String 
 * @default "mm/dd/yy"
 * @private
+% if ( RT->Config->Get('DateDayBeforeMonth') ) {
+*/var vDateDisplayFormat = "dd/mm/yy";
+% } else {
 */var vDateDisplayFormat = "mm/dd/yy";
+% }
 
 	  var vNumUnits  = 0;
       var vCaptionType;
@@ -1113,7 +1117,7 @@ Complete-Displays task percent complete</p>
          while(Date.parse(vTmpDate) <= Date.parse(vMaxDate))
          {	
             vStr = vTmpDate.getFullYear() + '';
-            vStr = vStr.substring(2,4);
+            // vStr = vStr.substring(2,4);
             
             
             if(vFormat == 'minute')
@@ -1132,25 +1136,20 @@ Complete-Displays task percent complete</p>
             
   	         if(vFormat == 'day')
             {
-			      vRightTable += '<td class=gdatehead style="FONT-SIZE: 12px; HEIGHT: 19px;" align=center colspan=7>';
-			     
-				 if(vDateDisplayFormat=='yyyy-mm-dd')
-				 {vRightTable += (vTmpDate.getMonth()+1) + '-' + (vTmpDate.getDate()) + ' - ';}
-				 else
-				 {vRightTable += JSGantt.formatDateStr(vTmpDate,vDateDisplayFormat.substring(0,5)) + ' - ';}
-				 
+			      vRightTable += '<td class=gdatehead style="FONT-SIZE: 12px; HEIGHT: 19px;" align=center colspan=7>' +
+			      JSGantt.formatDateStr(vTmpDate,vDateDisplayFormat.substring(0,5)) + ' - ';
                vTmpDate.setDate(vTmpDate.getDate()+6);
 		         vRightTable += JSGantt.formatDateStr(vTmpDate, vDateDisplayFormat) + '</td>';
                vTmpDate.setDate(vTmpDate.getDate()+1);
             }
             else if(vFormat == 'week')
             {
-  		         vRightTable += '<td class=gdatehead align=center style="FONT-SIZE: 12px; HEIGHT: 19px;" width='+vColWidth+'px>`'+ vStr + '</td>';
+  		         vRightTable += '<td class=gdatehead align=center style="FONT-SIZE: 12px; HEIGHT: 19px;" width='+vColWidth+'px>'+ vStr + '</td>';
                vTmpDate.setDate(vTmpDate.getDate()+7);
             }
             else if(vFormat == 'month')
             {
-	            vRightTable += '<td class=gdatehead align=center style="FONT-SIZE: 12px; HEIGHT: 19px;" width='+vColWidth+'px>`'+ vStr + '</td>';
+	            vRightTable += '<td class=gdatehead align=center style="FONT-SIZE: 12px; HEIGHT: 19px;" width='+vColWidth+'px>'+ vStr + '</td>';
                vTmpDate.setDate(vTmpDate.getDate() + 1);
                while(vTmpDate.getDate() > 1)
                {
@@ -1159,7 +1158,7 @@ Complete-Displays task percent complete</p>
             }
             else if(vFormat == 'quarter')
             {
-	            vRightTable += '<td class=gdatehead align=center style="FONT-SIZE: 12px; HEIGHT: 19px;" width='+vColWidth+'px>`'+ vStr + '</td>';
+	            vRightTable += '<td class=gdatehead align=center style="FONT-SIZE: 12px; HEIGHT: 19px;" width='+vColWidth+'px>'+ vStr + '</td>';
                vTmpDate.setDate(vTmpDate.getDate() + 81);
                while(vTmpDate.getDate() > 1)
                {
@@ -1247,14 +1246,25 @@ Complete-Displays task percent complete</p>
                   vWeekdayColor = "ffffff";
 
                if(vNxtDate <= vMaxDate) {
+
+% if ( RT->Config->Get('DateDayBeforeMonth') ) {
+                  vDateRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; HEIGHT: 19px; BORDER-LEFT: #efefef 1px solid;" bgcolor=#' + vWeekdayColor + ' align=center width:'+vColWidth+'px><div style="width: '+vColWidth+'px">' + vTmpDate.getDate() + '/'  + (vTmpDate.getMonth()+1) + '</div></td>';
+% } else {
                   vDateRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; HEIGHT: 19px; BORDER-LEFT: #efefef 1px solid;" bgcolor=#' + vWeekdayColor + ' align=center width:'+vColWidth+'px><div style="width: '+vColWidth+'px">' + (vTmpDate.getMonth()+1) + '/' + vTmpDate.getDate() + '</div></td>';
+
+% }
                   if( vCurrDate >= vTmpDate && vCurrDate < vNxtDate ) 
                      vItemRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; BORDER-LEFT: #efefef 1px solid;" bgcolor=#' + vWeekdayColor + ' align=center><div style="width: '+vColWidth+'px">&nbsp&nbsp</div></td>';
                   else
                      vItemRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; BORDER-LEFT: #efefef 1px solid;" align=center><div style="width: '+vColWidth+'px">&nbsp&nbsp</div></td>';
 
                } else {
+% if ( RT->Config->Get('DateDayBeforeMonth') ) {
+                  vDateRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; HEIGHT: 19px; BORDER-LEFT: #efefef 1px solid; bgcolor=#' + vWeekdayColor + ' BORDER-RIGHT: #efefef 1px solid;" align=center width:'+vColWidth+'px><div style="width: '+vColWidth+'px">' +  vTmpDate.getDate() + '/' + (vTmpDate.getMonth()+1) + '</div></td>';
+% } else {
                   vDateRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; HEIGHT: 19px; BORDER-LEFT: #efefef 1px solid; bgcolor=#' + vWeekdayColor + ' BORDER-RIGHT: #efefef 1px solid;" align=center width:'+vColWidth+'px><div style="width: '+vColWidth+'px">' + (vTmpDate.getMonth()+1) + '/' + vTmpDate.getDate() + '</div></td>';
+
+% }
                   if( vCurrDate >= vTmpDate && vCurrDate < vNxtDate ) 
                      vItemRowStr += '<td class="ghead" style="BORDER-TOP: #efefef 1px solid; FONT-SIZE: 12px; BORDER-LEFT: #efefef 1px solid; BORDER-RIGHT: #efefef 1px solid;" bgcolor=#' + vWeekdayColor + ' align=center><div style="width: '+vColWidth+'px">&nbsp&nbsp</div></td>';
                   else
@@ -1910,9 +1920,9 @@ JSGantt.folder= function (pID,ganttObj) {
             JSGantt.show(pID, 1, ganttObj);
 
                if (JSGantt.isIE()) 
-                  {JSGantt.findObj('group_'+pID).innerText = '-';}
+                  {JSGantt.findObj('group_'+pID).innerText = '–';}
                else
-                  {JSGantt.findObj('group_'+pID).textContent = '-';}
+                  {JSGantt.findObj('group_'+pID).textContent = '–';}
 
          }
 
@@ -1984,7 +1994,7 @@ JSGantt.show =  function (pID, pTop, ganttObj) {
          } else {
 
             if (JSGantt.isIE()) { // IE;
-               if( JSGantt.findObj('group_'+pID).innerText == '-') {
+               if( JSGantt.findObj('group_'+pID).innerText == '–') {
                   JSGantt.findObj('child_'+vID).style.display = "";
                   JSGantt.findObj('childgrid_'+vID).style.display = "";
                   vList[i].setVisible(1);
@@ -1992,7 +2002,7 @@ JSGantt.show =  function (pID, pTop, ganttObj) {
 
             } else {
 
-               if( JSGantt.findObj('group_'+pID).textContent == '-') {
+               if( JSGantt.findObj('group_'+pID).textContent == '–') {
                   JSGantt.findObj('child_'+vID).style.display = "";
                   JSGantt.findObj('childgrid_'+vID).style.display = "";
                   vList[i].setVisible(1);
@@ -2019,10 +2029,11 @@ JSGantt.taskLink = function(pRef,pWidth,pHeight)
 
   {
 
-    if(pWidth)  {vWidth =pWidth;}  else {vWidth =400;}
-    if(pHeight) {vHeight=pHeight;} else {vHeight=400;}
+//    if(pWidth)  {vWidth =pWidth;}  else {vWidth =400;}
+//    if(pHeight) {vHeight=pHeight;} else {vHeight=400;}
 
-    var OpenWindow=window.open(pRef, "newwin", "height="+vHeight+",width="+vWidth); 
+//    var OpenWindow=window.open(pRef, "newwin", "height="+vHeight+",width="+vWidth); 
+    var OpenWindow=window.open(pRef, "newwin"); 
 
   };
 
